nixos/services/rabbitmq/default.nix

```diff
@@ -283,20 +283,31 @@ in {
 
     flyingcircus.services = {
 
-      sensu-client.checks.rabbitmq-alive = {
-        notification = "rabbitmq amqp alive";
-        command = ''
-          ${pkgs.sensu-plugins-rabbitmq}/bin/check-rabbitmq-amqp-alive.rb \
-            -u fc-sensu -w ${cfg.listenAddress} -p ${sensuPassword}
-        '';
-      };
-
-      sensu-client.checks.rabbitmq-node-health = {
-        notification = "rabbitmq node healthy";
-        command = ''
-          ${pkgs.sensu-plugins-rabbitmq}/bin/check-rabbitmq-node-health.rb \
-            -u fc-sensu -w ${cfg.listenAddress} -p ${sensuPassword}
-        '';
+      sensu-client.checks = {
+        rabbitmq-alive = {
+          notification = "rabbitmq amqp alive";
+          command = ''
+            ${pkgs.sensu-plugins-rabbitmq}/bin/check-rabbitmq-amqp-alive.rb \
+              -u fc-sensu -w ${cfg.listenAddress} -p ${sensuPassword}
+          '';
+        };
+
+        rabbitmq-node-health = {
+          notification = "rabbitmq node healthy";
+          command = ''
+            ${pkgs.sensu-plugins-rabbitmq}/bin/check-rabbitmq-node-health.rb \
+              -u fc-sensu -w ${cfg.listenAddress} -p ${sensuPassword}
+          '';
+        };
+
+        rabbitmq-feature-flags-enabled = {
+          notification = "rabbitmq all stable feature flags enabled";
+          command = ''
+            ! /run/wrappers/bin/sudo -u rabbitmq ${pkgs.rabbitmq-server}/bin/rabbitmqctl list_feature_flags state stability |
```

Comment:
It's a bit a matter of taste. but I'd like to avoid hard-coded absolute paths when introducing new things.
`sudo` is in PATH in the check env.